.circleci/config.yml

```diff
@@ -163,6 +163,10 @@ jobs:
           name: Run e2e admin page test
           command: |
             python -m scripts.run_e2e_tests --skip-install --skip-build --suite="adminPage" --prod_env
+      - run:
+          name: Run e2e email dashboard page test
+          command: |
+            python -m scripts.run_e2e_tests --skip-install --skip-build --suite="emailDashboard" --prod_env
```

Comment:
Can you put this on github actions instead? There we can run tests faster because we get more parallelization

Reply:
@U8NWXD Should I create a new file or should I add it to a particular flow, because I don't think there is a file that in fithub workflow that is suitable for emailDashboard.

Reply:
Please create a new file

Reply:
You should remove it from CircleCI and add it to github actions. You'll also need to update the test file that checks that the config files are in sync to specify that your test runs on github actions

Reply:
Done.